doc/source/ray-core/patterns/limit-running-tasks.rst

```diff
@@ -0,0 +1,39 @@
+.. _core-patterns-limit-running-tasks:
+
+Pattern: Using resources to limit the number of concurrently running tasks
+==========================================================================
+
+In this pattern, we use :ref:`resources <resource-requirements>` to limit the number of concurrently running tasks.
+
+Running too many tasks at the same time might overload the cluster and cause issues like OOM.
+If that is the case, we can reduce the number of concurrently running tasks by increasing the amount of resources requested by those tasks.
+This works because Ray makes sure that the sum of the resource requirements of all of the concurrently running tasks on a given node cannot exceed the node's total resources.
```

Comment:
```suggestion
By default, Ray tasks require 1 CPU each, so the scheduler limits task concurrrency to the available CPUs.
Tasks that use more than 1 CPU (e.g., via mutlithreading) may experience slowdown due to interference from concurrent tasks, but otherwise are safe to run.

However, tasks that use more than their proportionate share of memory may overload a node and cause issues like OOM.
If that is the case, we can reduce the number of concurrently running tasks by increasing the amount of resources requested by those tasks.
This works because Ray makes sure that the sum of the resource requirements of all of the concurrently running tasks on a given node does not exceed the node's total resources.
```

Also add default number of CPUs for actors? I actually don't know what it is :D

Reply:
Actor is a bit special, it's 1 cpu for scheduling and 0 cpu for running.